core/lib/multivm/Cargo.toml

```diff
@@ -17,7 +17,7 @@ zk_evm_1_4_0.workspace = true
 zk_evm_1_3_3.workspace = true
 zk_evm_1_3_1.workspace = true
 vm2.workspace = true
-era_vm = {path = "../../../../"}
+era_vm = { path = "../../../../era_vm" }
```

Comment:
This will not work as a submodule, from era_vm, believe me I've tried. Please restore this.

Reply:
When developing, we can modify this as needed.